ui/pages/confirmations/components/confirm/info/personal-sign/siwe-sign/siwe-sign.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { toHex } from '@metamask/controller-utils';
+
+import { NETWORK_TO_NAME_MAP } from '../../../../../../../../shared/constants/network';
+import { useI18nContext } from '../../../../../../../hooks/useI18nContext';
+import { currentConfirmationSelector } from '../../../../../../../selectors';
+import { SignatureRequestType } from '../../../../../types/confirm';
+import {
+  ConfirmInfoRow,
+  ConfirmInfoRowAddress,
+  ConfirmInfoRowText,
+} from '../../../../../../../components/app/confirm/info/row';
+import { formatDate } from '../../../../../utils/date';
+
+const SIWESignInfo: React.FC = () => {
+  const t = useI18nContext();
+  const currentConfirmation = useSelector(
+    currentConfirmationSelector,
+  ) as SignatureRequestType;
+
+  if (!currentConfirmation.msgParams?.siwe?.parsedMessage) {
+    return null;
+  }
+
+  const siweMessage = currentConfirmation.msgParams.siwe?.parsedMessage;
+  const chainId = toHex(siweMessage.chainId);
+
+  return (
+    <>
+      <ConfirmInfoRow label={t('siweOriginalMessage')}>
```

Comment:
Should we assume we want alert support for all our rows and use `AlertRow`, as even if we don't yet have alerts, a future snap insight could add one?

Reply:
I would prefer to add alert to specific fields which would need it.

Reply:
My concern is how would we know which need it if the intent of snap insights is to add alerts to any field?

Reply:
it would then need adding alerts to all multiple fields of various confirmations - which is hard I think

Reply:
The properties are identical on `AlertRow` plus the need for `alertKey` and `ownerId`, so it's just a matter of replacing with `AlertRow` wherever we want to support alerts.

Reply:
ATM I have no idea which of these fields may need to be alerted by snaps, additionally though using `AlertRow` is easy but it brings along some more logic which I would prefer to add only if needed.

Reply:
It's not crucial in this PR, so we can review what fields we want to support and add in a later PR.